src/Microsoft.DotNet.VersionTools/Util/Command.cs

```diff
@@ -0,0 +1,349 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.VersionTools.Util
+{
+    class Command
```

Comment:
Yep, only `status`, `commit`, and `push`.

I tried removing the utility classes and putting what I needed in the static `Git` method, and came up with this: https://github.com/dotnet/buildtools/compare/master...dagood:simplify-versiontools-git. @ericstj Maybe it's more reasonable to include this than a full wrapper?